src/gluonts/block/encoder.py

```diff
@@ -246,27 +232,29 @@ def hybrid_forward(
         # return the last state as the static code
         static_code = F.slice_axis(ct, axis=1, begin=-1, end=None)
         static_code = F.squeeze(static_code, axis=1)
+
         return static_code, ct
 
 
 class RNNEncoder(Seq2SeqEncoder):
     """
-    Defines an RNN as the encoder.
+     Defines RNN encoder that uses covariates and target as input to the RNN if desired.
```

Comment:
```suggestion
    Defines RNN encoder that uses covariates and target as input to the RNN if desired.
```

Reply:
@lostella We could expose the scaler which is set to no operation scaling by default. Also I would remove the train method and from_inputs methods of the MQCNN for the GluonTS merge.